Plutarch/Api/V1.hs

```diff
@@ -25,7 +25,7 @@ module Plutarch.Api.V1 (
   type PStakeValidator,
 
   -- ** Value
-  Value.PValue (PValue),
```

Comment:
I need to remember to revert this

Reply:
did you forget to revert this

Reply:
Yes, we should make another PR to fix this